test/verify/check-journal

```diff
@@ -467,54 +478,136 @@ s.send(b"PRIORITY=3\\nFOO=bar\\n")'
-        sel = "#journal-entry-fields .nav .btn-danger"
-        b.wait_visible(sel)
-
-        sel = "#journal-entry-fields .nav .btn-primary:contains('Report')"
+        sel = "table.reporting-table tr:first-child .btn:contains('Report')"
         b.click(sel)
 
-        # jQuery magic on this page updates the whole frame for changing to "Reported" state
-        b.expect_load_frame("cockpit1:localhost/system/logs")
-        sel = "#journal-entry-fields .nav .problem-btn:contains('Reported')"
-        b.wait_visible(sel)
-        self.assertIn("/reports/bthash/123deadbeef", b.attr(sel, 'href'))
+        sel = "table.reporting-table tr:first-child a[href$='/reports/bthash/123deadbeef'"
+        b.wait_present(sel)
 
         # "Unreport" the problem to test reporting unknown problem
-        m.execute('find /var/spool/abrt -name "reported_to" | xargs rm')
+        m.execute('find /var/spool/abrt -name "reported_to" -or -name "ureports_counter" | xargs rm')
+        # The service also needs to be restarted, because the daemon maintains
+        # its own cache that interferes with resetting the state.
+        m.execute('systemctl restart reportd')
+        # We have no network access.
+        m.execute("sed -i 's|abrt-action-perform-ccpp-analysis|true|' /etc/libreport/events.d/ccpp_event.conf")
+        # reporter-bugzilla will not be run without a duphash file present.
+        m.execute("find /var/spool/abrt -depth -maxdepth 1 | head -1 | xargs -I {} cp '{}/uuid' '{}/duphash'")
 
         b.reload()
         b.enter_page("/system/logs")
 
-        sel = "#journal-entry-fields"
+        sel = "table.reporting-table tr:nth-child(2) button:contains('Report')"
+        b.click(sel)
+
+        test_user = 'correcthorsebatterystaple'
+
+        for purpose in ['text', 'password']:
+            b.wait_visible("#cockpit_modal_dialog .modal-content input[type='%s']" % (purpose))
+
+            sel = "#cockpit_modal_dialog .modal-content input"
+            b.set_val(sel, test_user)
+
+            sel = "#cockpit_modal_dialog .modal-footer button:contains('Send')"
+            b.click(sel)
+
+        sel = "#cockpit_modal_dialog .modal-content p:contains('Password')"
+        b.wait_not_present(sel)
+
+        sel = "#cockpit_modal_dialog .modal-footer button:contains('No')"
+        b.click(sel)
+
+        sel = "table.reporting-table tr:nth-child(2) a[href='https://bugzilla.example.com/show_bug.cgi?id=123456']"
         b.wait_present(sel)
 
-        self.allow_journal_messages('.*This problem has already been reported.')
-        self.allow_journal_messages('.*http://localhost:12345/reports/42/')
-        self.allow_journal_messages('.*https://bugzilla.example.com/show_bug.cgi\?id=123456')
+    @skipImage("ABRT does not work on i386", "fedora-i386")
+    @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable",
+               "ubuntu-1804", "fedora-atomic", "rhel-8-1", "rhel-8-1-distropkg", "rhel-8-2")
```

Comment:
This needs fedora-atomic → fedora-coreos, similar to the other testAbrt* in master now.